profiles/seccomp/default.json

```diff
@@ -1587,7 +489,165 @@
 					"valueTwo": 0,
 					"op": "SCMP_CMP_MASKED_EQ"
 				}
-			]
+			],
+			"comment": "",
+			"includes": {},
+			"excludes": {
+				"caps": [
+					"CAP_SYS_ADMIN"
+				],
+				"arches": [
+					"s390",
+					"s390x"
+				]
+			}
+		},
+		{
+			"names": [
+				"clone"
+			],
+			"action": "SCMP_ACT_ALLOW",
+			"args": [
+				{
+					"index": 1,
+					"value": 2080505856,
+					"valueTwo": 0,
+					"op": "SCMP_CMP_MASKED_EQ"
+				}
+			],
+			"comment": "",
```

Comment:
Can you add a comment here eg `"s390 parameter ordering for clone is different"` as we have comments!